client/src/app/components/employee-settings/employee-settings.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="edit-container">
+  <button mat-button (click)="editMode(!edit)">{{edit ? "Cancel": "Edit"}}</button>
+
+  <div *ngIf="employee">
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title>
+          {{edit? "Update": ""}} {{employee?.first_name + " " + employee?.last_name}}
+        </mat-card-title>
+      </mat-card-header>
+
+      <mat-card-content>
+
+          <mat-form-field>
+            <mat-label for="First Name"></mat-label>
+            <input matInput id="first_name" type="text" [disabled]="true" [value]="employee.first_name"/>
```

Comment:
Why can't we update the first name

Reply:
People's names don't change that easily

Reply:
Yeah i know but what if they made a typo.

Reply:
That would usually occur on account creation. If that's the case, they can always delete the user and create a new account.